pkgs/by-name/ko/koboldcpp/package.nix

```
@@ -105,15 +106,13 @@ effectiveStdenv.mkDerivation (finalAttrs: {
 
   env.NIX_LDFLAGS = lib.concatStringsSep " " (finalAttrs.darwinLdFlags ++ finalAttrs.metalLdFlags);
 
-  env.NIX_CFLAGS_COMPILE =
-    lib.optionalString (march != "") "-march=${march}" + lib.optionalString (mtune != "") "-mtune=${mtune}";
```

```suggestion
  env.NIX_CFLAGS_COMPILE =
    lib.optionalString (march != "")
      (lib.warn "koboldcpp: the march argument is only kept for compatibility; use overrideAttrs intead"
       "-march=${march}")
     + lib.optionalString (mtune != "")
       (lib.warn "koboldcpp: the mtune argument is only kept for compatibility; use overrideAttrs intead"
        "-mtune=${mtune}");
```

(excuse the formatting, I used the web interface...)